index.js

```diff
@@ -164,7 +165,7 @@ function LandroidAccessory(platform, name, serial, accessory) {
     this.accessory.getService(Service.BatteryService).getCharacteristic(Characteristic.StatusLowBattery).on('get', this.getStatusLowBattery.bind(this));
     this.accessory.getService(Service.BatteryService).getCharacteristic(Characteristic.ChargingState).on('get', this.getChargingState.bind(this));
 
-    this.accessory.getService(Service.ContactSensor).getCharacteristic(Characteristic.ContactSensorState).on('get', this.getContactSensorState.bind(this));
+    this.accessory.getService("ErrorSensor").getCharacteristic(Characteristic.ContactSensorState).on('get', this.getContactSensorStateError.bind(this));
```

Comment:
This line is the only problem, it expects that theres  Service called "ErrorSensor", that should be checked beforehand and the previous behavior should be used if thats not the case.